layouts/partials/fragments/nav.html

```diff
@@ -224,3 +224,10 @@
   });
 </script>
 {{- end -}}
+{{- if .Params.sticky }}
+<style>
+  :root {
+    --header-height: 56px;
+  }
+</style>
+{{- end -}}
```

Comment:
Not sure I like the hardcoding of that. Can't think of a better solution so.

Reply:
I can't think of a solution that doesn't involve JS and is not hard coding the value. I'll try to figure something out and maybe we can create an issue to track it.

Reply:
Fine to keep it as is and open a "revisit" to track this.